### DISCUSSION TITLE
Can a chart also be used as standalone Livewire component outside of the panel builder?

### DISCUSSION THREAD

**Discussion by django23** [leandrocfe/filament-apex-charts#49] [Q&A]
I am not using the panel builder at the moment, only Tables. 
But I would love to include a chart on one of the pages I created. 

Could that be done with this plugins?
Kind regards,

**Comment by leandrocfe** [comment]
Hi, Yes you can

**Reply by danie1net0** [reply]
How can I? In my case, when I try to use the component as a Filament Widget on a Livewire page, I get this error:

![image](https://github.com/leandrocfe/filament-apex-charts/assets/19168799/e0c49e0f-9c05-47ad-8205-2b526e624238)

My code is:
```
<x-layouts.dashboard :$title :$fullHeight>
  <div>
    @livewire(App\Livewire\BlogPostsChart::class)
  </div>
</x-layouts.dashboard>

**Comment by django23** [comment]
Sweet. I think my question if more about how to ;)

**Comment by hyphen81** [comment]
I have the same question. I've got it included in a non-dashboard page, but the chart isn't showing up.

**Comment by LukasJK** [comment]
Same - would love to use it outside of panels

**Reply by LukasJK** [reply]
@leandrocfe the issue seems to stem from this line of code
`$plugin = \Leandrocfe\FilamentApexCharts\FilamentApexChartsPlugin::get(); `
called in apex-chart-widget.blade.php - as far as I can tell it does nothing further down the line and removing it has no side-effects from what I could observe.

**Comment by leandrocfe** [comment]
I'll check it ✌️

**Comment by jaguarman** [comment]
Hi everybody,

Is it already possible to use a chart or chart widget in a standard filament page?
If so, it it possible to alter the size to max instead of a small widget or do I have to create a second widget only for using it outside the dashboard?

TIA
Peter
